build.gradle

```diff
@@ -2,7 +2,7 @@ buildscript {
     repositories {
         jcenter()
         maven {
-            url "https://plugins.gradle.org/m2/"
+            url = "https://plugins.gradle.org/m2/"
```

Comment:
`=` not needed.

Reply:
IntelliJ will show this line as an error/warning (Need to find which one).